docs/source/setup/gee.rst

```diff
@@ -1,174 +1,178 @@
 
-SEPAL recipes can be run from the main SEPAL interface using default SEPAL credentials or your own, personal GEE access credentials. However, to run the SEPAL applications that employ GEE, you will need to link your SEPAL and GEE accounts. SEPAL applications that make use of GEE will not run (e.g. authentication will not work) if your GEE and SEPAL accounts are unlinked.
+SEPAL recipes can be run from the main SEPAL interface using default SEPAL credentials or your personal GEE access credentials. However, to run the SEPAL applications that employ GEE, you will need to link your SEPAL and GEE accounts. SEPAL applications that make use of GEE will not run (e.g. authentication will not work) if your GEE and SEPAL accounts are unlinked.
 
 .. tip:
 
-   See linking SEPAL and GEE accounts in the previous section.
+   See linking SEPAL and GEE accounts in the previous section for more information.
 
-To get the best of SEPAL, we highly suggest all users to open their own Google Earth Engine account and link it to SEPAL. In this doc, we will guide you through the full process of creating a GEE account and link it to SEPAL. 
+To get the best of SEPAL, we highly suggest that all users open their own GEE account and link it to SEPAL. 
+
+In this section of SEPAL documentation, we will guide you through the full process of creating a GEE account and linking it to SEPAL. 
 
 Create a GEE account
 --------------------
 
 Sign-up
 ^^^^^^^
 
-Signing up for Google Earth Engine is required in order to properly export images and data products from SEPAL as GEE assets or SEPAL applications using GEE. You will need to have a Google Account in order to sign up. If you don't have one already, you can set one up here: https://accounts.google.com/servicelogin. 
+Signing up for GEE is required to properly export images and data products from SEPAL as GEE assets or SEPAL applications using GEE. 
+
+You will need to have a Google account to sign up. If you don't have one already, you can set one up here: https://accounts.google.com/servicelogin. 
 
 To request a GEE account, please visit https://earthengine.google.com/new_signup/.
 
 .. image:: ../_images/setup/register/gee_landing.png
-   :alt: Request access to google earth engine
+   :alt: Request access to google earth engine.
    :align: center
 
-Once you have a Google Earth Engine account, you can access GEE here: https://code.earthengine.google.com/.
+Once you have a GEE account, you can access the platform here: https://code.earthengine.google.com/.
 
 .. image:: ../_images/setup/register/gee_code.png
-   :alt: gee code editor
+   :alt: GEE code editor
```

Comment:
line 36 you added a "." at the end of the figure alt, but not here. When should I use one for figures legend ?